src/android/NativePageTransitions.java

```diff
@@ -196,8 +203,8 @@ public void run() {
           }
 
           if (href != null && !"null".equals(href)) {
-            if (!href.startsWith("#")) {
-              webView.loadUrlIntoView(webView.getUrl().substring(0, webView.getUrl().lastIndexOf('/')+1) + href, false);
+            if (!href.startsWith("#") && href.contains(".html")) {
+              webView.loadUrlIntoView(webView.getUrl().substring(0, webView.getUrl().indexOf("FlipletDataDir/")+15) + href, false);
```

Comment:
Suggest you use of a variable for this since you are using it in multiple places